Twig/Extension/SonataAdminExtension.php

```diff
@@ -36,18 +37,25 @@ class SonataAdminExtension extends \Twig_Extension
     protected $logger;
 
     /**
+     * @var TranslatorInterface|null
+     */
+    protected $translator;
+
+    /**
      * @var string[]
      */
     private $xEditableTypeMapping = array();
 
     /**
-     * @param Pool            $pool
-     * @param LoggerInterface $logger
+     * @param Pool                $pool
+     * @param LoggerInterface     $logger
+     * @param TranslatorInterface $translator
      */
-    public function __construct(Pool $pool, LoggerInterface $logger = null)
+    public function __construct(Pool $pool, LoggerInterface $logger = null, TranslatorInterface $translator = null)
```

Comment:
do we need to remove the `= null` in next major? if yes, please add a `NEXT_MAJOR`